package.json

```diff
@@ -28,8 +27,11 @@
     "debug": "~2.6.9",
     "dotenv": "^6.2.0",
     "express": "~4.16.0",
+    "express-jwt": "^5.3.1",
     "joi": "^14.3.1",
+    "jsonwebtoken": "^8.4.0",
     "morgan": "~1.9.0",
+    "nodemailer": "^5.1.1",
```

Comment:
What does this "nodemailer" do?

Reply:
It is used to send an email from a nodejs application(password rest link)